tests/unit/mocks.py

```diff
@@ -45,7 +47,7 @@ def __call__(
         assert args_tuple in self._commands, f'unhandled command {args}'
 
         returncode, stdout, stderr = self._commands[args_tuple]
-        self.calls.append(Call(args_tuple, returncode, stdout, stderr))
+        self.calls.append(Call(args_tuple, returncode, input, stdout, stderr))
```

Comment:
Speaking of keyword arguments, how about:
```suggestion
        self.calls.append(Call(args_tuple, returncode, stdin=input, stdout=stdout, stderr=stderr))
```

Reply:
Fair -- added.